include/library.h

```diff
@@ -194,6 +194,7 @@ class Library
     const std::string& language = "",
     const std::string& creator = "",
     const std::string& publisher = "",
+    bool other = false,
```

Comment:
Please use another name than "other".
"Other" make sense for the client of the library (kiwix-desktop) where we are looking for all other categories.
Here we are filtering book depending if the book has or has not the provided tags.
Something like "excludeTags" makes more sense (even if this is far than perfect, but all the api is crap)